src/components/periodic-report/migrations/Post.migration.ts

```diff
@@ -0,0 +1,72 @@
+import { BaseMigration, Migration } from '../../../core';
+
+@Migration('2021-08-03T15:30:05')
+export class PostMigration extends BaseMigration {
+  // :Post nodes have postMigration property in case of reversion
+  // using project.createdAt for post.modifiedAt –– it's an approximation, but not sure if we want to do that
+  async up() {
+    const before = await this.getPostCount(true);
+    this.logger.info(`${before ?? 0} 87Six comments before migration`);
+    await this.db
+      .query()
+      .raw(
+        ` 
+          match(u:User)-[:email { active: true }]->(:Property { value: "devops@tsco.org" })
+          match(p:Project)
+          with u.id as rootId, p,
+          [
+            { type: 'Note', shareability: 'Internal', body: p.commentDescription }, 
+            { type: 'Prayer', shareability: 'AskToShareExternally', body: p.commentPrayerNeeds }, 
+            { type: 'Note', shareability: 'Internal', body: p.commentProposalComments }
+          ] as posts
+          unwind posts as post
+          with rootId, p, post
+          where size(post.body) > 0
+          create 
+            (p)-[:post { active: true, createdAt: datetime() }]->(po:BaseNode:Post{ postMigration: true, createdAt: datetime(), id: apoc.create.uuid() }),
+              (po)-[:shareability { active: true, createdAt: datetime() }]->(:Property { createdAt: datetime(), value: post.shareability, sortValue: post.shareability }),
+              (po)-[:type { active: true, createdAt: datetime() }]->(:Property { createdAt: datetime(), value: post.type, sortValue: post.type }),
+              (po)-[:body { active: true, createdAt: datetime() }]->(:Property { createdAt: datetime(), value: post.body, sortValue: post.body }),
+              (po)-[:modifiedAt { active: true, createdAt: datetime() }]->(:Property { createdAt: datetime(), value: p.createdAt, sortValue: p.createdAt }),
+              (po)-[:creator { active: true, createdAt: datetime() }]->(:Property { createdAt: datetime(), value: rootId, sortValue: rootId })
+        `
+      )
+      .run();
+    const after = await this.getPostCount(false);
+    this.logger.info(`${after ?? 0} Cord V3 posts after migration`);
+
+    // this can be omitted if we want to keep data here in case of reversion
```

Comment:
This should be uncommented to remove 87Six comments, but I wasn't sure if we wanted to do that at the same time.

Reply:
Yes

Reply:
Yes to un-commenting it?

Reply:
I mean we should remove the old data